core/shutdown/channel.go

```diff
@@ -7,6 +7,14 @@ import (
 	"syscall"
 )
 
+var HardPanic bool
+
+func init() {
+	if os.Getenv("ENABLE_HARD_PANIC") == "true" {
+		HardPanic = true
+	}
+}
+
```

Comment:
Unused now?
```suggestion
```

Reply:
nope, its used in this file

Reply:
https://github.com/smartcontractkit/chainlink/pull/5384/files#diff-ad95a85a6c5931fa1b71b6341ccb7dd9659775e7f03ac9ebf17616ae2c27efcbR48

Reply:
Is that `Panic()` method unused?